pkg/skaffold/runner/runner.go

```diff
@@ -197,11 +191,13 @@ func getDeployer(runCtx *runcontext.RunContext) (deploy.Deployer, error) {
 	}
 }
 
-func getTagger(t latest.TagPolicy, customTag string) (tag.Tagger, error) {
+func getTagger(runCtx *runcontext.RunContext) (tag.Tagger, error) {
```

Comment:
so is the end goal here just to make everything consistent? maybe I'm mistaken but I thought that you wanted to move away from the runcontext in these constructors. just trying to understand

Reply:
:-) For now, I'm aiming at consistency.